packages/shared/examples/range-plot/range-plot.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component } from '@angular/core'
+import { Scale, Scatter } from '@unovis/ts'
+import { data, DataRecord, processLineData, LineDataRecord } from './data'
+
+const yScale = Scale.scalePoint([0, 800]).domain(data.map(d => d.occupation))
+@Component({
+  selector: 'range-plot',
+  templateUrl: './range-plot.component.html',
+})
+export class RangePlotComponent {
+  height = 1600
+  data = data
+  lineData = processLineData(data)
+  legendItems = [{ name: 'Women', color: '#FF6B7E' }, { name: 'Men', color: '#4D8CFD' }]
+  style = { width: '100%', height: this.height }
```

Comment:
This doesn't seem to be used in the template. I would just remove it since it is already declared there:
```
<vis-xy-container
  [width]="'100%'"
  [height]="height"
>
```